main/zoe/api/Zoe-Intro.md

```diff
@@ -19,32 +19,41 @@ This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide
 To start using a contract, you need to *install* it on Zoe. This is putting the contract’s code into Zoe. Multiple contracts can be installed on Zoe, and each contract can have multiple simultaneous active offers, each of which is handled by a separate contract instance. For clarity, in this document a *contract* is the installed code that defines how the contract works. A *contract instance* is an active execution of the contract code.
 
 
-Once installed on Zoe, a contract waits for an *offer* to be made. That starts a lifecycle for the offer and new contract instance of:
-1. Sending out *invites* to request counter-offers.
-2. Partie *redeeming* those invites and making a counter-offer.
-3. Determining when the offer *matches* a counter-offer.
-4. Either succeeding or failing to make the transaction. 
+Once installed on Zoe, a contract is instantiated, staring a lifecycle new contract instance of:
+1. Instantiating the contract with
+   ```js
+   const initialInviteP = zoe~.makeInstance(
+     installationHandle,
+     issuerKeywordRecord,
+     terms);
+   ```
+1. As shown above, `makeInstance` returns a promise for the initial invite, shown above as `initialInviteP`.
+1. Sometimes: obtaining the `publicAPI` from that initial invite, and then obtaining invites from that.
+1. Sending out invites to request offers (including counter-offers).
+1. Parties use those invites to make offers.
+1. Determining when an offer *matches* another offer.
```

Comment:
I think the concept of matching should be clearly restricted to Atomic Swap, so that users don't think that it is a Zoe concept.